core/src/edu/cornell/gdiac/main/GameController.java

```diff
@@ -120,6 +120,22 @@ public class GameController implements Screen, ContactListener {
    */
   private int countdown;
 
+
+  /**
+   * Counter for keep track of meter
+   */
+  private float metercounter;
+
+  /**
+   * Constant for when stationary
+   */
+  private float stationaryconst = 0.25f;
```

Comment:
Constants should be in SCREAMING_SNAKE_CASE, also all constants should be `final`